template-xml/data/src/main/java/co/nimblehq/template/xml/storage/NormalSharedPreferences.kt

```diff
@@ -1,4 +1,4 @@
-package co.nimblehq.template.data.storage
```

Comment:
Why do we remove `data` in data package?

Reply:
@luongvo Copied wrong 🙈 Thank to spotting this 🙏